[2022-07-18 23:26:54,964] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2009-01-10T06:00:00+00:00 [queued]>
[2022-07-18 23:26:54,989] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2009-01-10T06:00:00+00:00 [queued]>
[2022-07-18 23:26:54,991] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-07-18 23:26:54,992] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-07-18 23:26:54,994] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-07-18 23:26:55,019] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2009-01-10 06:00:00+00:00
[2022-07-18 23:26:55,035] {standard_task_runner.py:52} INFO - Started process 39301 to run task
[2022-07-18 23:26:55,052] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'local_to_gcs', 'local_to_gcs_task', 'scheduled__2009-01-10T06:00:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/local_to_gcs.py', '--cfg-path', '/tmp/tmpvnu03106', '--error-file', '/tmp/tmpl7x86ows']
[2022-07-18 23:26:55,056] {standard_task_runner.py:80} INFO - Job 53: Subtask local_to_gcs_task
[2022-07-18 23:26:55,146] {task_command.py:370} INFO - Running <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2009-01-10T06:00:00+00:00 [running]> on host 32d1f26135d5
[2022-07-18 23:26:55,271] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_to_gcs
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2009-01-10T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2009-01-10T06:00:00+00:00
[2022-07-18 23:27:01,872] {python.py:173} INFO - Done. Returned value was: None
[2022-07-18 23:27:01,922] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=local_to_gcs, task_id=local_to_gcs_task, execution_date=20090110T060000, start_date=20220718T232654, end_date=20220718T232701
[2022-07-18 23:27:01,991] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-18 23:27:02,021] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
